### ISSUE TITLE
Non HGVS conform output for hg38/ensembl

### ISSUE THREAD

**Issue by martenj** [charite/jannovar#388]
The HGVS annotation of the ensembl database is not sufficient. 
For the command: `
java -jar -Xmx4G jannovar.jar annotate-pos -d data/hg38_ensembl.ser --show-all -c "chr17:30995000C>T" `
you will get the following output:
```
#change	effect	hgvs_annotation	messages
chr17:30995000C>T	CODING_TRANSCRIPT_INTRON_VARIANT,CODING_TRANSCRIPT_INTRON_VARIANT,CODING_TRANSCRIPT_INTRON_VARIANT,CODING_TRANSCRIPT_INTRON_VARIANT,CODING_TRANSCRIPT_INTRON_VARIANT,CODING_TRANSCRIPT_INTRON_VARIANT	RNF135:ENST00000324689:c.517-2242C>T:p.(=),RNF135:ENST00000328381:c.680-2242C>T:p.(=),RNF135:ENST00000434242:c.95+1096C>T:p.(=),RNF135:ENST00000443677:c.373-2242C>T:p.(=),RNF135:ENST00000535306:c.744+1096C>T:p.(=),RNF135:ENST00000580444:c.469-2242C>T:p.(=)	.;.;.;.;.;.
```
where the trancript accession numbers are missing the release version, which is not HGVS annotation conform / recommondation. 
From this it's not clear which version of the transcript is used for the annotation calculation.
e.g. for ENST00000328381 

it is 
ENST00000328381.5 for Ensembl 75: Feb 2014 (GRCh37.p13)
ENST00000328381.6 for Ensembl 76: Aug 2014 (GRCh38) - gene set updated Jul 2014
ENST00000328381.7 for Ensembl 77: Oct 2014 (GRCh38) - patched/updated gene set Aug 2014
...
ENST00000328381.9 for the current Ensembl 91: Dec 2017 (GRCh38.p10)

**Comment by holtgrewe** [comment]
Do you see GRCh37 as also affected?

**Comment by holtgrewe** [comment]
Fixed in 2c8734dfb3bb0f91f44eef1bbc62efecc1e91c15 but only when transcript version is available (not for GRCh37...)
